config/htcondor-ce.spec

```diff
@@ -477,6 +480,7 @@ fi
 %config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-collector.conf
 %config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-collector-requirements.conf
 %config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-auth.conf
+%config(noreplace) %{_sysconfdir}/condor-ce/config.d/01-ce-collector-audit-payload.conf
```

Comment:
I'm not sure it should be in this sub-RPM?  I think you only want it in the above one.